docs/learn/learn-nft-pallets.md

```diff
@@ -24,7 +24,52 @@ on Westmint, which provides a multitude of functions to interact with NFTs.
 
 The pallet comes with a new way to configure NFTs, as well as configure collections and items.
 Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
-functionalities that are not needed in the runtime.
+functionalities not needed in the runtime.
+
+### Roles
+
+Setting up a collection implies different roles with different permissions:
+
+- Owner:
+
+  - destroy collection (to destroy the collection, there should be 0 items left).
+  - redeposit: re-evaluate the deposit on some items.
+  - set team: change the collection’s Issuer, Admin, Freezer.
+  - set collection max supply: set the maximum number of items for a collection.
+  - lock collection: this can include making a collection’s items non-transferable, fixing its max
+    supply, and locking collection metadata and attributes.
+
+- Admin:
+  - set attribute and metadata of a collection.
+  - set attributes pre-signed: set attributes for an item by providing the pre-signed approval.
+  - lock item properties: lock item metadata and attributes.
+
+:::info
+
+Note that an Admin account cannot burn or transfer items it does not own.
+
+:::
+
+- Freezer:
+
+  - lock item transfer: lock all holders from the possibility of transferring an item.
+  - unlock item transfer: lift a previous lock to transfer an item for all holders.
+
+- Issuer
+  - mint
+  - force mint (with custom item configuration): mint an item of a particular collection from a
+    privileged origin.
+  - mint pre-signed: mint an item by providing the pre-signed approval.
+  - update mint settings.
+
+For simple collections, the same account has all the roles by default. Still, for complex
+collections separate accounts can take each role with their responsibilities (e.g. items issuance).
+The key can be rotated for those roles. The owner's account is used to setup the collection, and its
+private key is kept in cold storage.
+
+Those roles can also be set to `none` without the ability to change them back. This is useful when a
```

Comment:
"Any role (Issuer/Admin/Freezer) can be set to `None` without the ability to change it back. This is useful when a"